### ISSUE TITLE
AttributeError: module 'tensorflow' has no attribute 'function'

### ISSUE THREAD

**Issue by mlchuxueing** [tensorflow/federated#395]
Hello,When i execute import tensorflow_federataed as tff,there exist a error:
AttributeError: module 'tensorflow' has no attribute 'function'.
tensorflow_federated vision:  0.4.0;
python vision: 3.5;
tensorflow vision :1.13.1;
Anyone can help me solve this question?

**Comment by ZacharyGarrett** [comment]
@mlchuxueing this looks like you're building TFF from source on the `master` branch; usage of `tf.function` was not part of the 0.4.0 release (was merged in f149d83399d7dc615a2efcfd5d728f4c1f706785). Is this correct?

If you'd like to continue using the source version before the next release, I believe you'll need to use `tf-nightly` TensorFlow pip package. On `master` we've moved to depend on the `tf-nightly` pip package (in 7cd281e6916638c865eb43dece888235f75cad94). The `requirements.txt` file has been updated to reflect this. Since you already have the `tensorflow` pip package installed, you may need to first uninstall it before installing `tf-nightly`. Something like:

  ```shell
  pip uninstall tensorflow
  pip install --upgrade -r tensorflow_federated/requirements.txt
  ```

**Comment by mlchuxueing** [comment]
> @mlchuxueing this looks like you're building TFF from source on the `master` branch; usage of `tf.function` was not part of the 0.4.0 release (was merged in [f149d83](https://github.com/tensorflow/federated/commit/f149d83399d7dc615a2efcfd5d728f4c1f706785)). Is this correct?
> 
> If you'd like to continue using the source version before the next release, I believe you'll need to use `tf-nightly` TensorFlow pip package. On `master` we've moved to depend on the `tf-nightly` pip package (in [7cd281e](https://github.com/tensorflow/federated/commit/7cd281e6916638c865eb43dece888235f75cad94)). The `requirements.txt` file has been updated to reflect this. Since you already have the `tensorflow` pip package installed, you may need to first uninstall it before installing `tf-nightly`. Something like:
> 
> ```shell
> pip uninstall tensorflow
> pip install --upgrade -r tensorflow_federated/requirements.txt
> ```

Okay， thank you!
